lib/events/encryptedevent.cpp

```diff
@@ -61,3 +61,10 @@ RoomEventPtr EncryptedEvent::createDecrypted(const QString &decrypted) const
     }
     return loadEvent<RoomEvent>(eventObject);
 }
+
+void EncryptedEvent::setRelation(const QJsonObject& relation)
+{
+    auto content = editJson()["content"_ls].toObject();
```

Comment:
You don't need `editJson()` here because you only actually edit it 2 lines below.
```suggestion
    auto content = contentJson();
```